### DISCUSSION TITLE
how to fix rpc error on requestTileTreeProps and generateTileContent

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#3908] [Q&A]
I'm writing a server use nestjs and view for snapshot viewing use BaseViewer component.

When my server starts, errors occured without detailed message on requestTileTreeProps and generateTileContent methods. It seems that these two are internal method of IModelTileRpcInterface and I don't know the actual useage.
![image](https://user-images.githubusercontent.com/19658624/177897469-37f6cfa3-db74-4f4c-84d5-8c0670f8a51c.png)

As for the frontend, a CORS error also occurs on the InitializeInterface.
![image](https://user-images.githubusercontent.com/19658624/177897758-73d2aaac-4846-4f84-8e25-45bfc9fc9a1c.png)

How to fix these errors?

**Comment by aruniverse** [comment]
Hi. The InitializeInterface issue has been fixed with iTwin.js 3.2.7, please try updating to that version. If you are still running into issues with the requesTileTreeProps and generateTileContent please let us know.
